client/common/Button.tsx

```diff
@@ -181,69 +249,7 @@ const Button = ({
   );
 };
 
-Button.defaultProps = {
```

Comment:
it's so great that we can get rid of all of this extra code at the bottom of the React files now haha, thanks for doing this conversion!!